src/cbmc/cbmc_parse_options.cpp

```diff
@@ -839,6 +832,19 @@ bool cbmc_parse_optionst::process_goto_program(
     // add loop ids
     goto_functions.compute_loop_numbers();
 
+    // label the assertions
+    label_properties(goto_functions);
```

Comment:
I fixed this one and I added comments to explain the current behaviour regarding labelling properties.